js/server/augmentLMData.js

```diff
@@ -1,37 +1,39 @@
-const _ = require('lodash');
-const moment = require('moment');
-const { START_DATETIME } = require('./config');
+const _ = require("lodash");
+const moment = require("moment");
+const { START_DATETIME } = require("./config");
 
-exports.augmentLMData = data => {
+exports.augmentLMData = (data) => {
   const users = _.uniq(
-    _.flatten(data.map(timestamp => Object.keys(timestamp.users)))
+    _.flatten(data.map((timestamp) => Object.keys(timestamp.users)))
   );
 
-  data.forEach(timestamp => {
+  data.forEach((timestamp) => {
     const timestampTotalTickets = _.sum(
-      _.map(timestamp.users, user => {
-        return _.sum(user.tickets.map(t => t.amount));
+      _.map(timestamp.users, (user) => {
+        return _.sum(user.tickets.map((t) => t.amount));
       })
     );
-    timestamp.totalTickets = timestampTotalTickets;
-    timestamp.users = _.forEach(timestamp.users, user => {
-      const totalTickets = _.sum(user.tickets.map(t => t.amount));
-      user.claimableReward =
-        user.claimed + _.sum(user.tickets.map(t => t.reward * t.mul));
+    timestamp.totalTicketsAmountSum = timestampTotalTickets;
+    timestamp.users = _.forEach(timestamp.users, (user) => {
```

Comment:
I concur. (not my code) Intellisense said it returns something like `LodashIterator`. But I'm not fluent in lodash and I was trying to make as few changes as possible to the core logic

Reply:
> Can you give a summary of the main logic changes here? Since you moved stuff to the user.js file (good thing), hard to keep track of the diff.

Also, on second look, I didn't actually make any changes to the logic in `augmentLMData`. Just renaming of properties. Much of the code there could utilize the `User` prototype. But LM is working as desired, so we should probably leave it as-is until that changes